.github/workflows/benchmark.yml

```diff
@@ -85,6 +85,7 @@ jobs:
         # In benchmark, we use local as master so set driver memory only. Note that GitHub Actions has 7 GB memory limit.
         bin/spark-submit \
```

Comment:
I think it's better to set the `SPARK_HOME` env instead. Can you add:

```
SPARK_HOME: ${{ github.workspace }}
```

under `env` with a comment that explains why we should set it?

Reply:
Thanks for the review. Updated it and retrigger the benchmark workflow in my fork.
https://github.com/pingsutw/spark/actions/runs/997492502